rpi_backlight.py

```diff
@@ -195,7 +291,10 @@ def gui() -> None:
 
     win = Gtk.Window(title="Set display brightness")
 
-    ad1 = Gtk.Adjustment(value=get_actual_brightness(), lower=11, upper=255)
+    if mode == "MODE_TINKERBOARD":
+        ad1 = Gtk.Adjustment(value=get_brightness_value(), lower=0, upper=255)
+    elif mode == "MODE_RPI":
+        ad1 = Gtk.Adjustment(value=get_brightness_value(), lower=11, upper=255)
```

Comment:
Again, since the min value is now 0 there's no need for the distinction.

Reply:
fixed